migrations/2022-06-12-012121_add_site_hide_modlog_names/up.sql

```diff
@@ -0,0 +1 @@
+alter table site add column hide_modlog_mod_names boolean default true NOT NULL;
```

Comment:
So do we want to remove this option all together? It works for me either way.

Reply:
IMO yes, but I wanted to wait to hear what @Nutomic has to say first. I just can't think of a situation where you'd want mod names public under any circumstances.

Reply:
Not all communities deal with much harrassment. In those cases, it might be simpler to see who did the mod action, and have a conversation with that mod directly. Especially because we have no such thing like mod mail. As its already coded, i would keep the functionality, instead of extra effort to remove and reimplement later.